index.html

```diff
@@ -1,98 +1,142 @@
-<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<!-- Don't shim CSS Custom Properties on IE11 -->
+		<script>
+			if (!window.Promise) {
+				window.ShadyCSS = { nativeCss: true };
+			}
+		</script>
 
-<script src="/packages/core/lib/core.umd.js"></script>
-<script src="/packages/components/root/lib/root.umd.js"></script>
-<script src="/packages/components/button/lib/button.umd.js"></script>
-<script src="/packages/components/button-group/lib/button-group.umd.js"></script>
+		<!-- Enable ES5 class-less Custom Elements -->
+		<script src="/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+		<!-- Load appropriate polyfills and shims -->
+		<script
+			src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"
+			defer
+		></script>
 
-<link rel="stylesheet" href="/packages/core/src/vars.css"><script> </script>
-<link rel="stylesheet" href="/index.css">
-</head>
-<body>
-	<ts-root>
-		<header slot="header" class="header">Header</header>
-		<footer slot="footer" class="footer">Footer</footer>
-		<section slot="sidebar-left" class="sidebar-left">Left Sidebar</section>
-		<section slot="sidebar-right" class="sidebar-right">Right Sidebar</section>
-		<section slot="sidebar-inner-left" class="sidebar-inner-left">Inner Left Sidebar</section>
-		<section slot="sidebar-inner-right" class="sidebar-inner-right">Inner Right Sidebar</section>
+		<!-- Set :root CSS Custom Properties -->
+		<link rel="stylesheet" href="/packages/core/src/vars.css" />
+		<!-- Set @font-face for all needed fonts -->
+		<link rel="stylesheet" href="/packages/core/src/fonts.css" />
+		<!-- Set margin/padding to be 0 for <html> -->
+		<link rel="stylesheet" href="/packages/core/src/reset.css" />
 
-		<article id="ts-root-toggle">
-			<h2>ts-root</h2>
-			<ts-button-group>
-				<ts-button data-toggle="header">Toggle Header</ts-button>
-				<ts-button data-toggle="footer">Toggle Footer</ts-button>
-				<ts-button data-toggle="sidebar-left">Toggle Left Sidebar</ts-button>
-				<ts-button data-toggle="sidebar-right">Toggle Right Sidebar</ts-button>
-				<ts-button data-toggle="sidebar-inner-left">Toggle Inner Left Sidebar</ts-button>
-				<ts-button data-toggle="sidebar-inner-right">Toggle Inner Right Sidebar</ts-button>
-			</ts-button-group>
-			<script>
-				function toggleSlot(name, button) {
-					var enabledSlot = document.querySelector('[slot=' + name + ']');
-					if (enabledSlot) {
-						button.removeAttribute('type');
-						return enabledSlot.setAttribute('slot', '!' + name);
-					}
+		<!-- Load user-styles -->
+		<link rel="stylesheet" href="/index.css" />
 
-					var disabledSlot = document.querySelector('[slot="!' + name + '"]');
-					if (disabledSlot) {
-						button.setAttribute('type', 'primary');
-						return disabledSlot.setAttribute('slot', name);
-					}
-				}
+		<!-- Load Tradeshift Elements once the polyfills are ready (optional?) -->
+		<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (18) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)